arangod/RestHandler/RestImportHandler.cpp

```diff
@@ -1070,7 +1072,7 @@ void RestImportHandler::createVelocyPackObject(
   VPackArrayIterator itKeys(keys);
   VPackArrayIterator itValues(values);
  
-  if (itKeys.size() != itValues.size()) { 
+  if (!_ignoreMissing && itKeys.size() != itValues.size()) { 
```

Comment:
It looks like we would not ignore only missing documents. We would ignore additional documents as well.